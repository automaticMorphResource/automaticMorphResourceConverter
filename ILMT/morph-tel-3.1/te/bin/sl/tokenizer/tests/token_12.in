ప్రపంచంలో ఉండే ఇలాంటి రీఫ్ లలో ఇరవై శాతం మేరకు ఆస్ట్రేలియా చుట్టూ ఉండే సముద్ర ప్రాంతాలలోనే ఉండటం విశేషంగా చెప్పవచ్చు. 
