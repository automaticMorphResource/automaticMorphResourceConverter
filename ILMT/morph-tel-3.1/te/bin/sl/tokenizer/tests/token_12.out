<Sentence id="1">
1	ప్రపంచంలో	unk
2	ఉండే	unk
3	ఇలాంటి	unk
4	రీఫ్	unk
5	లలో	unk
6	ఇరవై	unk
7	శాతం	unk
8	మేరకు	unk
9	ఆస్ట్రేలియా	unk
10	చుట్టూ	unk
11	ఉండే	unk
12	సముద్ర	unk
13	ప్రాంతాలలోనే	unk
14	ఉండటం	unk
15	విశేషంగా	unk
16	చెప్పవచ్చు	unk
17	.	unk
</Sentence>

